modules/ai/assets/js/editor/components/ai-promotion-infotip-content.js

```diff
@@ -0,0 +1,37 @@
+import {
+	Button,
+	Card,
+	CardActions,
+	CardContent,
+	CardHeader,
+	Typography,
+} from '@elementor/ui';
+import { __ } from '@wordpress/i18n';
+
+const AiPromotionInfotipContent = ( props ) => {
+	return (
+		<Card sx={ { width: 300 } }>
```

Comment:
```suggestion
		<Card sx={ { width: 300 } } elevation={ 0 }>
```
@DennisNerush  - to fix the color difference between the box and the arrow of the tooltip.